src/components/UserSelector.tsx

```diff
@@ -74,7 +63,7 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(setSelectedUser(user));
               }}
```

Comment:
Might be, but I suggest to move callback into separated function if you need to write it with more than 1 line